operator/src/main/java/org/bf2/operator/operands/KafkaInstance.java

```diff
@@ -88,6 +93,9 @@ public OperandReadiness getReadiness(ManagedKafka managedKafka) {
             // TODO: it may be a good idea to offer a message here as well
             return new OperandReadiness(isDeleted(managedKafka) ? Status.False : Status.Unknown, Reason.Deleted, null);
         }
+        if (managedKafka.getAnnotation(ManagedKafkaKeys.Annotations.PAUSE_RECONCILIATION).map(Boolean::valueOf).orElse(false)) {
+            return new OperandReadiness(Status.Unknown, Reason.Paused, null);
```

Comment:
Unsure on whether a `Reason` should be set here - thoughts?

Reply:
The reason is good.  A message would be good as well.

Reply:
Added a message - happy to make adjustments if it can be reworded better.